api/auth.py

```
@@ -483,4 +483,98 @@ async def delete_api_key(
+    """Create a test user with predefined credentials (admin only)"""
+    # Check admin key
+    expected_admin_key = os.getenv("ADMIN_SECRET_KEY")
+    if not expected_admin_key or admin_key != expected_admin_key:
+        raise HTTPException(
+            status_code=status.HTTP_403_FORBIDDEN,
+            detail="Invalid admin key"
+        )
+    
+    # Define test user credentials
+    test_email = "test@trendit.dev"
+    test_password = "TestPassword123"
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "test_password"

(S105)